tools/inference/lightning_inference.py

```diff
@@ -76,12 +74,15 @@ def infer():
     )
 
     # create the dataset
-    dataset = InferenceDataset(args.input, image_size=tuple(config.dataset.image_size), transform=transform)
+    dataset = InferenceDataset(
+        args.input, image_size=tuple(config.dataset.image_size), transform=transform  # type: ignore
```

Comment:
I get this `mypy` error otherwise
```
Argument "image_size" to "InferenceDataset" has incompatible type "Tuple[Any, ...]"; expected "Union[int, Tuple[int, int], None]"
```